web/models/log.ex

```diff
@@ -15,8 +15,8 @@ defmodule Gannbaruzoi.Log do
   end
 
   def first_of(task_id) do
-    Repo.get Task, task_id
-    |> Ecto.assoc(:logs)
+    task = Repo.get Task, task_id
```

Comment:
`Repo.one from l in __MODULE__, where: l.task_id == ^task_id, limit: 1`
fixed with this!

this generates,
`SELECT l0."id", l0."task_id", l0."inserted_at", l0."updated_at" FROM "logs" AS l0 WHERE (l0."task_id" = $1) LIMIT 1 [1]`